website_sale_vat_required/static/src/js/website_sale_vat_required.tour.js

```diff
@@ -1,40 +1,40 @@
 /* License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
  */
-
 odoo.define('website_sale_vat_required.tour', function(require) {
-
     "use strict";
 
-    require("website_sale.tour");
```

Comment:
I didn't check if this changed in v13, but in general terms do not remove this. You are subject to a race condition without it.